.github/workflows/backend-lint.yml

```
@@ -0,0 +1,22 @@
+name: backend-lint
+
+on:
+  push:
+  pull_request:
+    branches: [ main ]
```

I think this is backwards. If we have
```suggestion
  push:
    branches: [ main ]
  pull_request:
```
then it only trigggers the push when the branch is main (force pushing to main), and otherwise the pull_request trigger handles synchronization events for in-PR pushes.

---

ah yep confirmed this fixes my double action running!